docs/modules/ROOT/pages/ios_settings.adoc

```diff
@@ -71,9 +71,9 @@ image:84_Settings_themes.png[84_Settings_themes.png]
 |Dark theme
 |Light theme
 
-a|image:themes-ownCloud-Classic.png[ownCloud iOS App - Classic theme]
-a|image:themes-ownCloud-Dark.png[ownCloud iOS App - Dark theme]
-a|image:themes-ownCloud-Light.png[ownCloud iOS App - Light theme]
+a|image:themes/classic.png[ownCloud iOS App - Classic theme]
+a|image:themes/dark.png[ownCloud iOS App - Dark theme]
+a|image:themes/light.png[ownCloud iOS App - Light theme]
 |===
```

Comment:
Would be good, to describe what the happens, when theme `System Appearance` is enabled and what this means.

Reply:
Can you provide more information about that, please?

Reply:
When `System Appearance` is selected, ownCloud will automatically use the selected iOS system theme (light or dark) to reflect the system UI. Setting `System Appearance` is only available up from iOS 13.